### PR TITLE
Fix for APIMANAGER-3682

### PR DESCRIPTION
**Pull request by thilinicooray**
Fix for APIMANAGER-3682 : Add breadcrumbs or back button in “edit content view” In API Documentation

Added breadcrumb in in-line editor view

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/documentation/inline-editor/template.jag [hunk 1]
```diff
@@ -1,5 +1,13 @@
 <% jagg.template("documentation/inline-editor", function(inputs, outputs, jagg) {
 var mode = request.getParameter("mode");
+var infoMappedUrl=jagg.getMappedUrl("/site/pages/item-info.jag");
+var tenantDomain= encode.forUriComponent(request.getParameter("tenant"));
+var urlPostfix;
+if(tenantDomain!='null'){
+    urlPostfix="&tenant="+tenantDomain;
+}else{
+    urlPostfix='';
```
Comment (jranabahu) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/documentation/inline-editor/template.jag]: needs proper formatting

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-info/js/load-tabs.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ $(document).ready(function() {
             var clickedTab = e.target.href.split('#')[1];
             ////////////// edit tab
             pushDataForTabs(clickedTab);
-            $.cookie("selectedTab",clickedTab);
+            $.cookie("selectedTab", clickedTab,  {path: "/"});
```
Comment (jranabahu) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-info/js/load-tabs.js]: These are minor issues but please make sure to be consistent. There are 2 spaces between ,  {path: "/"}

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/documentation/inline-editor/template.jag [hunk 3]
```diff
@@ -1,5 +1,13 @@
 <% jagg.template("documentation/inline-editor", function(inputs, outputs, jagg) {
 var mode = request.getParameter("mode");
+var infoMappedUrl=jagg.getMappedUrl("/site/pages/item-info.jag");
+var tenantDomain= encode.forUriComponent(request.getParameter("tenant"));
+var urlPostfix;
+if(tenantDomain!='null'){
+    urlPostfix="&tenant="+tenantDomain;
+}else{
+    urlPostfix='';
```
Comment (jranabahu) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/documentation/inline-editor/template.jag]: Needs formatting

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-info/js/load-tabs.js [hunk 4]
```diff
@@ -64,7 +64,7 @@ $(document).ready(function() {
             var clickedTab = e.target.href.split('#')[1];
             ////////////// edit tab
             pushDataForTabs(clickedTab);
-            $.cookie("selectedTab",clickedTab);
+            $.cookie("selectedTab", clickedTab,  {path: "/"});
```
Comment (jranabahu) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-info/js/load-tabs.js]: Please be consistent. Remove the additional space. :)

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/documentation/inline-editor/template.jag (hunk 1 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/documentation/inline-editor/template.jag]: Fixed in PR #1033 commit 8fc6c10

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/documentation/inline-editor/template.jag (hunk 3 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/documentation/inline-editor/template.jag]: Fixed in PR #1033 commit 8fc6c10

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-info/js/load-tabs.js (hunk 2 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/item-info/js/load-tabs.js]: Fixed in PR #1033 commit 8fc6c10

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-info/js/load-tabs.js (hunk 4 above)
Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/item-info/js/load-tabs.js]: Fixed in PR #1033 commit 8fc6c10

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/documentation/inline-editor/js/inline.js [hunk 5]
```diff
@@ -71,3 +71,8 @@ function saveContent(provider, apiName, apiVersion, docName, mode) {
 var hideMsg=function () {
     $('#docAddMessage').hide("fast");
 }
+
+function navigateBack(tabName){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/documentation/inline-editor/js/inline.js]: need formatting

Comment (thilinicooray) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/responsive/templates/documentation/inline-editor/js/inline.js]: Fixed in PR #1280

**Pull request merged**